### PR TITLE
New: [NA] [Style] [Content] Changed Title HTML default value and styling [ED-10884]

### PR DESCRIPTION
**Pull request by Barcov**

### CODE REVIEW

**Review by ArmanPurtoian — DISMISSED** [review]

File: modules/nested-accordion/widgets/nested-accordion.php [hunk 1]
```diff
@@ -245,8 +245,23 @@ protected function register_controls() {
...
+				'selectors_dictionary' => [
+					'h1' => '--n-accordion-title-font-size: 2.5rem;',
+					'h2' => '--n-accordion-title-font-size: 2rem;',
+					'h3' => '--n-accordion-title-font-size: 1,75rem;',
+					'h4' => '--n-accordion-title-font-size: 1.5rem;',
+					'h5' => '--n-accordion-title-font-size: 1rem;',
+					'h6' => '--n-accordion-title-font-size: 1.rem; ',
+					'div' => '--n-accordion-title-font-size: 1.rem;',
+					'span' => '--n-accordion-title-font-size: 1.rem; ',
+					'p' => '--n-accordion-title-font-size: 1.rem;',
```
Comment (louiswol94) [inline modules/nested-accordion/widgets/nested-accordion.php:257]:
```suggestion
					'h6' => '--n-accordion-title-font-size: 1rem; ',
					'div' => '--n-accordion-title-font-size: 1rem;',
					'span' => '--n-accordion-title-font-size: 1rem; ',
					'p' => '--n-accordion-title-font-size: 1rem;',
```

File: modules/nested-accordion/assets/scss/frontend.scss [hunk 2]
```diff
@@ -34,21 +33,27 @@
...
+					margin-block: 0;
+				}
 
 				&-icon {
 					display: flex;
 					flex-direction: row;
 					align-items: center;
 					padding-inline: var( --n-accordion-title-icon-padding-inline );
 					order: var( --n-accordion-title-icon-order );
+					overflow: hidden;
```
Comment (louiswol94) [inline modules/nested-accordion/assets/scss/frontend.scss:54]: @Barcov Why do we need `overflow:hidden`?

**Review by louiswol94 — CHANGES_REQUESTED** [review]

File: modules/nested-accordion/widgets/nested-accordion.php (hunk 1 above)
Comment (Barcov) [reply modules/nested-accordion/widgets/nested-accordion.php]: Nice catch

File: modules/nested-accordion/assets/scss/frontend.scss (hunk 2 above)
Comment (Barcov) [reply modules/nested-accordion/assets/scss/frontend.scss]: Agreed

**Review by louiswol94 — APPROVED** [review]

**Pull request merged**